### PR TITLE
ref: first stubs

### PR DESCRIPTION
**Pull request by casperdcl**
Just stub pages with a little description for now. The idea (for now) is *not* to duplicate `cml {cmd} --help` output but instead provide examples, edge cases, warnings, etc.

- part of #55
- part of https://github.com/iterative/cml/issues/514

### CODE REVIEW

File: content/docs/ref/pr.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+```dvc
+SHA="$(git log -n1 --format=%h)"
+BASE="$(git branch)"
+git checkout -b "${BASE}-cml-pr-${SHA}"
+git add *.py *.json
+git commit -m "CML PR for ${SHA} [skip ci]"
+git push
+gh pr create -B ${BASE}
+git checkout -
```
Comment (casperdcl) [inline content/docs/ref/pr.md:19]:
```suggestion
```

Just re-looked at the implementation and this seems to be removed now?

**Review by 0x2b3bfa0 — APPROVED** [review]

File: content/docs/ref/send-comment.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+```
+
+## Common error messages
+
+### Bitbucket
+
+- Can't create a pull request comment: the Pull Request Commit Links application
+  has not been installed.
+
+  - We don't like ClickOps either but here's an actual
```
Comment (0x2b3bfa0) [inline content/docs/ref/send-comment.md:14]: Minor: does it have to be a nested bullet point? 🤔

File: content/docs/ref/runner.md [hunk 3]
```diff
@@ -0,0 +1,5 @@
+# Command Reference: `runner`
+
+Starts a runner (either via any supported cloud compute provider or locally
+on-premise -- see
+[cloud compute resource credentials](/doc/self-hosted-runners#cloud-compute-resource-credentials)).
```
Comment (0x2b3bfa0) [inline content/docs/ref/runner.md:5]: Minor: perhaps a more general link to the top of `/doc/self-hosted-runners` instead?

File: content/docs/ref/pr.md (hunk 1 above)
Comment (0x2b3bfa0) [reply content/docs/ref/pr.md:19]: Yes, it doesn't work like that anymore (if it ever did), and it [might not be what users expect](https://discord.com/channels/485586884165107732/728693131557732403/888777744220049478)

Comment (0x2b3bfa0) [reply content/docs/ref/pr.md:19]: @casperdcl, it's probably material for a separate issue, but the short answer is that it currently remains on the newly created branch. 🙃

File: content/docs/ref/send-comment.md (hunk 2 above)
Comment (casperdcl) [reply content/docs/ref/send-comment.md:14]: unnecessary future-proofing indeed

**Pull request merged**